discovery-provider/src/api/v1/playlists.py

```diff
@@ -139,14 +140,52 @@ def get(self, playlist_id):
         playlist_id = decode_with_abort(playlist_id, full_ns)
         args = current_user_parser.parse_args()
         current_user_id = get_current_user_id(args)
-        playlist = get_playlist(playlist_id, current_user_id)
+        get_playlist_args = {
+            "with_users": True,
+            "current_user_id": current_user_id,
+            "playlist_id": playlist_id,
+        }
+        playlist = get_playlist(get_playlist_args)
         if playlist:
             tracks = get_tracks_for_playlist(playlist_id, current_user_id)
             playlist["tracks"] = tracks
         response = success_response([playlist] if playlist else [])
         return response
 
 
+@full_ns.route("/by_permalink/<string:handle>/<string:slug>")
+class FullPlaylistByHandleAndSlug(Resource):
+    @ns.doc(
+        id="""Get Playlist By Handle and Slug""",
+        description="""Get a playlist by handle and slug""",
+        params={"handle": "playlist owner handle", "slug": "playlist slug"},
+    )
+    @ns.expect(current_user_parser)
+    @ns.marshal_with(full_playlists_response)
+    @cache(ttl_sec=5)
+    def get(self, handle, slug):
+        args = current_user_parser.parse_args()
+        current_user_id = get_current_user_id(args)
+
+        get_playlist_args = {
+            "with_users": True,
+            "current_user_id": current_user_id,
+        }
+
+        route = {
+            "handle": handle,
+            "slug": slug,
+        }
+        get_playlist_args["routes"] = [route]
```

Comment:
does this need to be a list?

Reply:
i'm just following the implementation of playlist ids in the get_playlists function, which takes in a list of playlist_ids. it feels weird to me to have one argument be a list and the other be a single item especially when the query is called `get_playlists` and not `get_playlist`